guide/payments/request.md

```diff
@@ -0,0 +1,477 @@
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/Home.png"
+   retina = "/assets/images/guide/payments/request/Home@2x.png"
+   layout = "shadow"
+   caption = "Users should be able to access their profile via the home page."
+   alt-text = "Screen showing home screen number pad entry."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/my-profile-empty.png"
+   retina = "/assets/images/guide/payments/request/my-profile-empty@2x.png"
+   layout = "shadow"
+   caption = "A users profile is a great place to generate a new Lightning address."
+   alt-text = "Screen showing users profile."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/generate-lightning-address.png"
+   retina = "/assets/images/guide/payments/request/generate-lightning-address@2x.png"
+   layout = "shadow"
+   caption = "Make generating a Lightning address simple."
+   alt-text = "Screen showing generation of a Lightning address."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/my-profile.png"
+   retina = "/assets/images/guide/payments/request/my-profile@2x.png"
+   layout = "shadow"
+   caption = "A newly generated reusable Lightning address used to receive regular payments to."
+   alt-text = "Screen showing newly generated Lightning address."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/contact-list.png"
+   retina = "/assets/images/guide/payments/request/contact-list@2x.png"
+   layout = "shadow"
+   caption = "Contact lists are great for storing and managing reusable payment requests."
+   alt-text = "Screen showing contact list."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/contact-card.png"
+   retina = "/assets/images/guide/payments/request/contact-card@2x.png"
+   layout = "shadow"
+   caption = "Contact cards can store the users reusable payment requests."
+   alt-text = "Screen showing users contact card."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/contact-card.png"
+   retina = "/assets/images/guide/payments/request/contact-card@2x.png"
+   layout = "shadow"
+   caption = "Contact cards can store reusable payment requests of other users too."
+   alt-text = "Screen showing another users contact card."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+## Withdrawal request
+
+Another way a user can request bitcoin is from a withdrawal request. A withdrawal request is a payment request that lets the receiver withdraw bitcoin from another user's wallet. Unlike previously mentioned payment requests, withdrawal requests are generated by the sender. These can be useful for merchants providing refunds to customers.
```

Comment:
```suggestion
Another way a user can request bitcoin is from a withdrawal request. A withdrawal request is a payment request that lets the receiver withdraw bitcoin from another wallet. Unlike previously mentioned payment requests, withdrawal requests are generated by the sender. These can be useful for merchants providing refunds to customers.
```
Removed `user`, as it could also be a merchant, a web service, etc.